beacon-chain/rpc/validator/exit.go

```diff
@@ -20,8 +21,10 @@ func (vs *Server) ProposeExit(ctx context.Context, req *ethpb.SignedVoluntaryExi
 	}
 
 	// Confirm the validator is eligible to exit with the parameters provided.
-	err = exit.ValidateVoluntaryExit(s, vs.GenesisTime, req)
-	if err != nil {
+	if int(req.Exit.ValidatorIndex) >= len(s.Validators) {
```

Comment:
What if req.Exit is nil here?

Reply:
I don't think that is possible from gRPC, but i'll add a check anyway